src/game_state/GameCamera.h

```diff
@@ -0,0 +1,36 @@
+// Braden Beck
+// Modified: Matthew Stewart - 1/28/17
+
+#ifndef GAME_CAMERA_H_
+#define GAME_CAMERA_H_
+
+#include <memory>
+
+#include "MatrixStack.h"
+
+#define CAMERA_SPEED 0.4
+#define DEGREES_IN_CIRCLE 360.0f
+#define VIEWING_ANGLE 160.0f
+
+class GameCamera {
+public:
+   GameCamera();
+   GameCamera(glm::vec3 eyePos, glm::vec3 lookAtPos, glm::vec3 up);
+
+   MatrixStack getView();
+   MatrixStack pivot(int width, int height, double xpos, double ypos);
+
+private:
+   MatrixStack ViewMatrix = MatrixStack();
+   glm::vec3 eyePos = glm::vec3(0,0,0);
```

Comment:
Just a nit, but it would be nice if the variable names used the same formatting. In the style guide we are loosely following, they would be snake case instead of pascal/camel case